src/material-experimental/mdc-slider/slider-thumb.ts

```diff
@@ -47,18 +47,36 @@ export interface MatSliderDragEvent {
 }) export class MatSliderThumb {
   /** The current value of this slider input. */
   @Input()
-  get value(): number { return this._value; }
-  set value(v: number) { this._value = v; }
-  private _value: number;
+  get value(): number { return Number(this._elementRef.nativeElement.getAttribute('value')); }
+  set value(v: number) {
+    this._initialized = true;
+
+    // If the foundation has already been initialized, we need to
+    // relay any value updates to it so that it can update the UI.
+    if (this._slider._initialized) {
+      this._slider._setValue(v, this.thumb);
+    } else {
+      // Setup for the MDC foundation.
+      this._elementRef.nativeElement.setAttribute('value', v.toString());
```

Comment:
This doesn't quite make sense to me. You set up a binding for `value` in the host bindings, but you also set up this getter/setter to manually update. In general with Angular, you would treat the DOM as _output_ of the component and not treat is as the source of truth for any state, letting the binding keep the DOM up-to-date based on the internal state of the component.